app/src/main/java/com/example/cattlelog/MainActivity.kt

```diff
@@ -114,6 +178,11 @@ class MainActivity : AppCompatActivity() {
 
     private fun updateDatabaseAvailabilityStatus() {
         if (targetDatabaseFile.exists()) {
+            // TODO: use `checkDatabaseVersion()` to find the date of the most recent database, compare
+            //  it against current version.  On initial database download, we should also create a
+            //  timestamp file using the date string from `checkDatabaseVersion()`.  Then, all we need
+            //  to do to determine if the database needs to be updated is compare the string stored locally
+            //  against the result from the `checkDatabaseVersion()` function.
             databaseStatusTextView.setText(getString(R.string.already_have_database))
```

Comment:
Same here regarding rebasing. I removed this in #11, though maybe we should've kept it around? Idk.

We can issue a toast notification instead so we don't clutter the UI in the meantime.

Reply:
Removed, but we do need to add the functionality from that TODO